tripy/tripy/flat_ir/flat_ir.py

```diff
@@ -152,8 +153,13 @@ def to_mlir_impl():
             outfile.flush()
             outfile.seek(0)
             stderr = outfile.read()
-
-            map_error_to_user_code_and_raise(self, exc, stderr.decode())
+            
+            # Capture the full traceback
+            exc_type, exc_value, exc_traceback = sys.exc_info()
+            traceback_str = ''.join(traceback.format_exception(exc_type, exc_value, exc_traceback))
+            
+            # Pass the combined error info to the error mapping function
+            map_error_to_user_code_and_raise(self, exc, f"Exception Traceback:\n{traceback_str}\n")
```

Comment:
I don't think this is right. We need to provide `stderr` because the MLIR-TRT error message that contains the location is emitted there. The Python traceback doesn't include enough information

Reply:
Yeah, missed passing the previously provided `stderr`.

Reply:
```
====================================== 1968 passed, 73 skipped, 1 deselected, 1 warning in 629.87s (0:10:29) ======================================
```